gazelle/python/BUILD.bazel

```diff
@@ -29,22 +29,20 @@ go_library(
     importpath = "github.com/bazel-contrib/rules_python/gazelle/python",
     visibility = ["//visibility:public"],
     deps = [
-        "//manifest",
```

Comment:
It seems like gazelle itself wasn't run on the gazelle module recently? Was there a reason for these unused deps?

Reply:
I don't know, TBH. If things are building and testing fine without them then... it's probably OK? :laughing: 

Looks like the dep was dropped in #2746, so somewhat recently. And I approved the PR :facepalm: